mantle/kola/harness.go

```diff
@@ -632,6 +636,21 @@ func runProvidedTests(tests map[string]*register.Test, patterns []string, multip
 	return firstRunErr
 }
 
+func serializeRerunData() error {
+	file, err := os.Create(filepath.Join(Options.CosaWorkdir, "tmp/kola/rerun-data"))
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	writer := bufio.NewWriter(file)
+	for _, test := range failedTests {
+		fmt.Fprintln(writer, test.Name)
+	}
+
+	return writer.Flush()
+}
```

Comment:
We already keep a report in JSON format in `tmp/kola/$platform-latest/report/report.json`. Let's reuse that instead?

Reply:
I can try that route.